client/components/payment-activity/types.ts

```diff
@@ -1,4 +1,11 @@
 export interface DateRange {
+	key: string; // Key
```

Comment:
Might be helpful to add an example so clear to reader.

```suggestion
	key: string; // Key, eg. last_7_days
```

Reply:
See above, let's give this a name that makes the purpose clear, e.g. `preset_name` or similar. `key` is very generic.

Reply:
+1 I think it would add clarity if we named this `preset_name` rather than `key`.

Reply:
Also, rather than `// comments like this` comments, using `/** comments like this */` is a good way to write comments for TS interfaces → helpful context for future code readers.

```ts
export interface DateRange {
	/** The name of the date range preset */
	preset_name: string;
  /** The date range start datetime used to calculate transaction data, e.g. 2024-04-29T16:19:29 */
	date_start: string;
  /** The date range end datetime used to calculate transaction data, e.g. 2024-04-29T16:19:29 */
	date_end: string; // End date
}
```

Reply:
Thanks! I have implemented it here: [d909a63](https://github.com/Automattic/woocommerce-payments/pull/8927/commits/d909a63934d1e23fa51bbc2906b369bff6c0ecb7) . Note that the interface definition has now moved within hooks.ts

Reply:
Moved to [hooks.ts](https://github.com/Automattic/woocommerce-payments/blob/cdfb1ebae0b6594f4eee841bec652daf93de0c7a/client/components/payment-activity/hooks.ts#L10) and fixed.

Reply:
> Also, rather than // comments like this comments, using /** comments like this */ is a good way to write comments for TS interfaces → helpful context for future code readers.

TIL @Jinksi ! To clarify, is the `/** */` style required for IDE type hint tooling to work properly? If so, might be a good best practice guideline :)

Reply:
😀 In this case, we mention this as a guideline in [docs/typescript/react-components.md](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/react-components.md#declare-the-attributes-on-the-component-with-an-interface), and [docs/typescript/general-conventions.md](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/general-conventions.md#use-jsdoc-to-provide-relevant-documentation).

<img width="1063" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/1d8bc82d-4293-49e5-a4e2-b2c251664d9b">

<img width="1066" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/2593cf11-c83a-4493-929d-703f6e0b526f">